plugins/playlist-backend/src/permissions/DefaultPlaylistPermissionPolicy.test.ts

```diff
@@ -67,11 +67,10 @@ describe('DefaultPlaylistPermissionPolicy', () => {
       resourceType: PLAYLIST_LIST_RESOURCE_TYPE,
       conditions: {
         anyOf: [
-          playlistConditions.isOwner([
-            'user:default/me',
-            'group:default/owner',
-          ]),
-          playlistConditions.isPublic(),
+          playlistConditions.isOwner({
+            owners: ['user:default/me', 'group:default/owner'],
+          }),
+          playlistConditions.isPublic({}),
```

Comment:
hmm - it would be nice to support the "no params" use case without having to pass an empty object, do you think it's worth exploring sprinkling some `| undefined` around the types to see if we can make that work?

Reply:
@mtlewis and I looked into this together and a bit separately. It's not so easy to do with the way the types are at the moment, so while it's not the greatest API experience we've decided to postpone this for the moment. We can revisit at a later date and improve this, likely without a breaking change.